shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/check/datasource/AbstractDataSourceCheckerTest.java

```diff
@@ -64,19 +64,41 @@ public void checkVariable(final Collection<? extends DataSource> dataSources) {
             }
         };
         dataSources = new LinkedList<>();
+        RuleAlteredJobContext jobContext = new RuleAlteredJobContext(ResourceUtil.mockJobConfig());
+        dataSource = new PipelineDataSourceManager().getDataSource(jobContext.getTaskConfig().getDumperConfig().getDataSourceConfig());
         dataSources.add(dataSource);
```

Comment:
1, Could we create `dataSource` by `PipelineDataSourceCreator`, but not `RuleAlteredJobContext`.

2, It's better to close `dataSource` or `PipelineDataSourceManager` after unit test.